javascript/javascript2/week1/danish_letters.js

```diff
@@ -0,0 +1,21 @@
+// Find and count the Danish letters
+const danishString = 'Jeg har en blå bil';
+const danishString2 = 'Blå grød med røde bær';
+
+function sumDanishLetters(str) {
+    let result = { total: 0 };
+    for (let i = 0; i < str.length; i++) {
+        if (str[i] === 'å' || str[i] === 'ø' || str[i] === 'æ') {
```

Comment:
to be super fancy you could do the following here:
`if(['å', 'ø', 'æ'].some(x => x===str[i]))`